plugins/backend/catalog-backend-module-aws/src/utils/tags.test.ts

```diff
@@ -58,68 +59,85 @@ describe('labelsFromTags and ownerFromTags', () => {
   });
 
   describe('ownerFromTags', () => {
+    const groups = [
+      { kind: 'Group', metadata: { name: 'owner1', namespace: 'test' } },
+      { kind: 'Group', metadata: { name: 'owner2', namespace: 'default' } },
+    ] as Entity[];
+
     it('should return "unknown" if no tags are provided', () => {
       const result = ownerFromTags();
       expect(result).toBe('unknown');
     });
 
-    it('should return owner from array of tags', () => {
+    it('should return owner from array of tags if the owner is found in existing groups', () => {
       const tags = [
         { Key: 'owner', Value: 'owner1' },
         { Key: 'tag:two', Value: 'value2' },
       ];
+      const result = ownerFromTags(tags, undefined, groups);
+      expect(result).toBe('group:test/owner1');
+    });
+
+    it('should not return owner from array of tags if the owner is not found in existing groups', () => {
+      const tags = [
+        { Key: 'owner', Value: 'owner3' },
+        { Key: 'tag:two', Value: 'value2' },
+      ];
+      const result = ownerFromTags(tags, undefined, groups);
+      expect(result).toBe('unknown');
```

Comment:
would it make more sense  to default this to the plain text value instead of unknown in this case?

Reply:
Yes could do but the owner won't relate to a catalog entity then. Maybe that's fine

Reply:
yeah, I was thinking that the unknown won't likely match catalog either but at least the attempted value could give some information what was the initial wanted value.

Reply:
Yea true. I changed it to that now